.github/workflows/governor-token-list.yml

```diff
@@ -0,0 +1,61 @@
+name: Governor Token List Update
+on:
+  schedule:
+    - cron: '0 0 1,15 * *' # Run at 00:00 on the 1st and 15th of every month
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  token-list-update:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.20.10"
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - name: Generate new token list
+        env:
+          PRICE_TOLERANCE: "50" # Price tolerance in %. Feel free to change this number.
+        run: |
+          cd node/hack/governor
+          npm ci
+          npm run start
+      - name: Verify new token list
+        run: |
+          cd node/hack/governor
+          go run check_query.go
+      - name: Run Governor tests
+        run: |
+          cd node/pkg/governor
+          go test
+      # IF UPSTREAMING IN THE FUTURE, REMOVE THE NEXT 2 STEPS
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v9
```

Comment:
Please use commit hashes for external (non-github official actions). Default git [lightweight tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_creating_tags) in git are mutable, and in the case of [Endbug/add-and-commit@v9](https://github.com/EndBug/add-and-commit/tree/v9) they update the tag with the latest v9.x.

Hypothetically speaking, this could be used to insert a malicious change in the PR pipeline. It is best to pin these by commits like [wormhole does](https://github.com/wormhole-foundation/wormhole/blob/bf9660f75bdf14c50746d66a63735e098f9cd988/.github/workflows/build.yml#L124).

I also recommend, if you have cycles, take a skim through the code of external actions before using them.

Reply:
Thanks again for the reminder! Have updated both the `add-and-commit` action and the `create-pull-request` action to use specific commits (aligning with the last stable release)